### DISCUSSION TITLE
请求修正已被封禁账号无法通过Cookies登陆的问题

### DISCUSSION THREAD

**Discussion by RMWW** [AdNotFound/EhViewer#108] [Ideas]
目前彩E和绿E均已修复了此问题，希望这款白E也能修复好此问题，以方便公共账号使用。

之前彩E的相关修正：
https://github.com/UjuiUjuMandan/EhViewer/releases/tag/1.11.6-RC2-cc
https://github.com/UjuiUjuMandan/EhViewer/releases/tag/1.11.7.2-cc

**Comment by AdNotFound** [comment]
最新版本已解决

**Comment by RMWW** [comment]
辛苦了白E越发完美了
